app/assets/javascripts/application.js

```diff
@@ -1,2 +1,4 @@
 //= require turbolinks
-//= require app/utils
+
+import 'app/utils';
+import 'app/pw-toggle';
```

Comment:
Hmm... I don't think the best practices here.   @monfresh does this looks sensible?  Is the "turbolinks" require above still needed?